docs/source/subprojects.rst

```diff
@@ -0,0 +1,136 @@
+.. _subprojects:
+
+
+===================
+Jupyter subprojects
+===================
+
+Project Jupyter is developed as a set of subprojects. The following is a topical list of the
+officially supported and maintained subprojects with links to the documentation or GitHub repo of
+each.
+
+
+Jupyter applications
+--------------------
+
+The Jupyter Applications create a foundation of interactive computing environments
+where scientific computing, data science, and analytics can be performed using
+a wide range of programming languages.
+
+.. glossary::
+
+
+    `Jupyter Notebook <http://jupyter-notebook.readthedocs.org/en/latest/>`__
+        Web-based application for authoring documents that combine live-code
+        with narrative text, equations and visualizations.
+
+    `Jupyter Console <https://github.com/jupyter/jupyter_console>`__
+        Terminal based console for interactive computing.
+
+    `Jupyter QtConsole <https://github.com/jupyter/qtconsole>`__
+        Qt application for interactive computing with rich output.
+
+
+Kernels
+-------
+
+Kernels are `programming language specific` processes that run independently
+and interact with the Jupyter Applications and their user interfaces. `IPython <http://ipython.org>`__ is the reference Jupyter kernel, providing a
+powerful environment for interactive computing in Python.
+
+.. glossary::
+
+    `IPython <http://ipython.org>`__
+        interactive computing in Python.
+
+    `ipywidgets <https://github.com/ipython/ipywidgets>`__
+        interactive widgets for Python in the Jupyter Notebook.
+
+    `ipyparallel <http://ipyparallel.readthedocs.org/en/latest/>`__
+        lightweight parallel computing in Python offering seamless notebook integration.
+
+.. seealso::
+
+   `Jupyter kernels <https://github.com/ipython/ipython/wiki/IPython-kernels-for-other-languages>`_
+      A full list of kernels available for other languages
```

Comment:
Can you precise that not all kernel might be functional  and write explicitely that they are not officially supported ?